src/ophyd_async/panda/panda.py

```diff
@@ -30,6 +30,8 @@ class SeqBlock(Device):
 
 class PcapBlock(Device):
     active: SignalR[bool]
+    arm: SignalX
+    disarm: SignalX
```

Comment:
This is modelled as a single SignalRW in the IOC:
https://github.com/PandABlocks/PandABlocks-ioc/blob/acfefb3da61db47a32d6fff71f26236c35909812/src/pandablocks_ioc/_pvi.py#L95-L104

Did you have a go with this against a real PandA with a the PandA IOC? It should have failed to `connect()` saying it can't find `arm` and `disarm` in the pvi structure...

```suggestion
    arm: SignalRW[bool]
```